.cirun.yml

```diff
@@ -3,9 +3,9 @@ runners:
   - name: win11-23h2-pro-arm64-16
     cloud: azure
     instance_type: Standard_D16plds_v5
-    machine_image: "/subscriptions/88c2ce23-b441-4d79-8f1c-50d9bc95ed08/resourceGroups/Win-CI/providers/Microsoft.Compute/galleries/base_images/images/win11-23h2-pro-arm64/versions/2024.04.22"
+    machine_image: "/subscriptions/88c2ce23-b441-4d79-8f1c-50d9bc95ed08/resourceGroups/Win-CI/providers/Microsoft.Compute/galleries/base_images/images/win11-23h2-pro-arm64/versions/2024.05.07"
     labels:
-      - cirun-win11-23h2-pro-arm64-16-2024-04-22
+      - cirun-win11-23h2-pro-arm64-16-2024-05-07
```

Comment:
In the future should we drop the date suffix from this label so that each workflow file doesn't need to be updated?

Reply:
If the suffix was not present, everytime we update the image there's no guarantee which runner picks it up. It is mostly the case with idle runners but still, there's a subtle race condition that can happen.

Reply:
Is that ok?

Reply:
Yeah, that makes sense!